src/components/TopNav/constants.ts

```diff
@@ -0,0 +1,18 @@
+import type { NavLink } from "./types";
+
+export const primaryNavLinks: NavLink[] = [
+  { href: "/users", text: "Users" },
+  {
+    href: "/services",
+    text: "Services",
+    children: [{ href: "/kliento", text: "Kliento" }],
+  },
+  { href: "/overview", text: "Tech overview" },
+  { href: "/about", text: "About" },
+];
+
+export const secondaryNavLinks: NavLink[] = [
+  { href: "/kliento/clients", text: "Clients", id: "clients" },
+  { href: "/kliento/servers", text: "Servers", id: "servers" },
+  { href: "/kliento/overview", text: "Overview", id: "overview" },
+];
```

Comment:
You could/should nest the secondary nav items. With the current approach, it'd be impossible to have another section with its own secondary navigation. For example:

```suggestion
export const primaryNavLinks: NavLink[] = [
  { href: "/users", text: "Users" },
  {
    href: "/services",
    text: "Services",
    children: [{
      href: "/kliento",
      text: "Kliento",
      children: [{
        { href: "/clients", text: "Clients", id: "clients" },
        { href: "/servers", text: "Servers", id: "servers" },
        { href: "/overview", text: "Overview", id: "overview" },
        }],
    }],
  },
  { href: "/overview", text: "Tech overview" },
  { href: "/about", text: "About" },
];
```

However, I don't mind this too much because it works, so if you don't want to do it or it's too much work, don't do it.

Reply:
I think it makes sense to nest the Kliento secondary navigation items under the Kliento item., which is itself under 'services'. With this approach we'll be able to add another section with its own secondary navigation in the future.